next.config.js

```diff
@@ -46,8 +46,7 @@ const config = {
     OKTA_CLIENT_ID: process.env.OKTA_CLIENT_ID ?? '0oa1n0gjoy3j5Ycdg0h8',
     OKTA_CLIENT_SECRET: process.env.OKTA_CLIENT_SECRET,
     OKTA_ISSUER: process.env.OKTA_ISSUER ?? 'https://signon.okta.com',
-    OKTA_SIGNOUT_REDIRECT_URL:
-      process.env.OKTA_SIGNOUT_REDIRECT_URL ?? 'https://mpdx.org/logout',
+    OKTA_SIGNOUT_REDIRECT_URL: process.env.OKTA_SIGNOUT_REDIRECT_URL ?? siteUrl,
```

Comment:
We're dropping the `/logout` path from the default?

Reply:
It's no longer needed. This is what was happening before:
1. When we sign out, we log out of Okta locally.
2. Then we send the user to `https://signon.okta.com/login/signout? fromURI${process.env.OKTA_SIGNOUT_REDIRECT_URL}` which logs the user out of the Okta session.
3. It then returns the user to the `process.env.OKTA_SIGNOUT_REDIRECT_URL` location which was pointed to the old MPDX logout page to ensure we log the user out of both applications.
4. Upon navigating to mpdx.org/logout, we would also log out of Angular as well as React.

Since the old app is no more, we no longer need it to be redirected to the old MPDX to log out the user. Instead, we return the user to the login screen. mpdx.org/login.

Reply:
And then if there isn't a `OKTA_SIGNOUT_REDIRECT_URL` defined, we return it to the homepage of MPDX.